sdcm/sct_events/loaders.py

```diff
@@ -28,34 +28,43 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class GeminiEvent(BaseStressEvent, abstract=True):
-    error: Type[SctEventProtocol]
-    warning: Type[SctEventProtocol]
-    start: Type[SctEventProtocol]
-    finish: Type[SctEventProtocol]
+class GeminiStressEvent(BaseStressEvent):
 
-    def __init__(self, cmd: str, result: Optional[Result] = None, severity: Severity = Severity.NORMAL):
+    def __init__(self, node: Any,
+                 cmd: str,
+                 log_file_name: Optional[str] = None,
+                 result: Optional[Result] = None,
+                 severity: Severity = Severity.NORMAL):
         super().__init__(severity=severity)
 
+        self.node = str(node)
         self.cmd = cmd
+        self.log_file_name = log_file_name
         self.result = ""
+        self.add_result(result)
```

Comment:
why not  use next only
```
self.add_result(result)
```
and update method `add_result` return "" if param result is None

Reply:
fixed